dali/test/dali_test_matching.h

```diff
@@ -70,11 +72,11 @@ class GenericMatchingTest : public DALISingleOpTest<ImgType, OutputImgType> {
     }
   }
 
-  inline DALIOpType OpType() const                { return m_nOpType; }
-  inline void setOpType(DALIOpType opType)        { m_nOpType = opType; }
+  inline OpType GetOpType() const                { return m_nOpType; }
+  inline void SetOpType(OpType opType)        { m_nOpType = opType; }
 
 
-  DALIOpType m_nOpType = DALI_GPU;
+  OpType m_nOpType = OpType::GPU;
```

Comment:
Please get rid of `m_n` prefix. First of all `n` should be for `number of` which is not the case here.